app/views/application/head/_favicons.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= favicon_link_tag asset_path("/favicon.svg"), rel: 'icon', type: 'image/svg+xml' %>
+<%= favicon_link_tag asset_path("/favicon.png"), rel: 'icon', type: 'image/png' %>
```

Comment:
We were previously serving only the favicon.png but it was being done from the asset_path. I added that helper back in and moved PNG into that directory. Is that okay? I assume so.

I also updated the SVG so it would have a transparent background in order to match the PNG. But I did not know how to do this quickly from the ICO file so I simply deleted it. Do you think we need to have an ICO file as well?

Reply:
I was encountering a minor issue with regard to the favicon in the asset_path. When using Safari, if you click the "share" button and then look at the terminal output you should see routing errors like ` ActionController::RoutingError (No route matches [GET] "/favicon.png")` and `ActionController::RoutingError (No route matches [GET] "/favicon.svg")` despite the files existing in the asset path. 
I do not think we need the ico file. I used a figma plugin to generate the icons and it was included, so I dropped it in there.